src/AccessibilityInsights.Extensions.GitHub/ConfigurationModelControl.xaml.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationModel.xaml
+    /// </summary>
+    public partial class ConfigurationModel : IssueConfigurationControl
+    {
+        public string Config { get; set;}
+        public ConfigurationModel()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Called when save button clicked.
+        /// </summary>
+        /// <returns>The extension’s new configuration, serialized</returns>
+        public override string OnSave()
+        {
+            this.Config = this.tbURL.Text;
```

Comment:
I wonder if we should save a version similar to what we do in ai-win's configuration. If we ever want to support an additional property it might make the serialization/deserialization easier.